mllib/src/main/scala/org/apache/spark/ml/feature/Binarizer.scala

```diff
@@ -112,7 +112,7 @@ final class Binarizer @Since("1.4.0") (@Since("1.4.0") override val uid: String)
         (Seq($(inputCol)), Seq($(outputCol)), Seq($(threshold)))
       }
 
-    val ouputCols = inputColNames.zip(tds).map { case (inputColName, td) =>
+    val mappedOutputCols = inputColNames.zip(tds).map { case (inputColName, td) =>
```

Comment:
There's an `outputCols` thing a few lines above, the oddly named variable here accidentally didn't collide with it...